en/interview-questions/basic-level.md

```diff
@@ -839,6 +839,59 @@ JavaScript is an interpreted language, not a compiled language. An interpreter i
 
 Yes, JavaScript is a case sensitive language. The language keywords, variables, function & object names, and any other identifiers must always be typed with a consistent capitalization of letters.
 
+## 6.Miscellaneous
```

Comment:
I think we have some repetitive content here. Can you fix it?

Reply:
@sumn2u I have removed the same content.Please review it